frontend/app/equity/dividend_computations/page.tsx

```
@@ -0,0 +1,178 @@
+function DividendComputationsContent() {
+  const [computations, setComputations] = useState(mockComputations);
+
+  useEffect(() => {
+    // Load demo computations from localStorage
+    const demoComputations = JSON.parse(localStorage.getItem('demoComputations') || '[]');
+    const allComputations = [...mockComputations, ...demoComputations];
+    setComputations(allComputations);
+  }, []);
+
+  const handleDelete = (computationId: number) => {
+    if (confirm('Are you sure you want to delete this computation?')) {
```

ESLint

[error] Unexpected confirm.

(no-alert)